scripts/change-log-generator.js

```diff
@@ -329,40 +373,43 @@ function writeChangeLog(textToInsert) {
   fs.writeSync(fd, buffer, 0, buffer.length, 0);
   fs.writeSync(fd, data, 0, data.length, buffer.length);
   fs.closeSync(fd);
+  console.log(`\nChange log written to: ${constants.CHANGE_LOG_PATH}`);
 }
 
 function openPRForChanges(releaseBranch, changeLogBranch) {
+  if (ADD_VERBOSE_LOGGING) {
+    console.log('\nOpening Pull Request for team review.');
+  }
   const commitCommand = `git commit -a -m "chore: generated CHANGELOG for ${releaseBranch}"`;
   const pushCommand = `git push origin ${changeLogBranch}`;
   shell.exec(commitCommand);
   shell.exec(pushCommand, { silent: true });
   shell.exec(
-    `open "https://github.com/forcedotcom/salesforcedx-vscode/pull/new/${changeLogBranch}"`
+    `open "https://github.com/forcedotcom/salesforcedx-vscode/pull/new/${releaseBranch}...${changeLogBranch}"`
   );
 }
 
 function writeAdditionalInfo() {
-  if (ADD_VERBOSE_LOGGING) {
-    console.log('\nStep 6: Write results to the change log.');
-  }
-  console.log(`Change log written to: ${constants.CHANGE_LOG_PATH}`);
   console.log('\nNext Steps:');
-  console.log("  1) Remove entries that shouldn't be included in the release.");
-  console.log('  2) Add documentation links as needed.');
+  console.log('  1) Remove entries that are not customer facing.');
+  console.log(
+    '  2) Add documentation links: [Doc Title](https://forcedotcom.github.io/salesforcedx-vscode/articles/doc-link-here)'
+  );
+  console.log(
+    '  3) Add external contributors: Contribution by [@contributor](https://github.com/contributor)'
+  );
   console.log(
-    '     Format: [Doc Title](https://forcedotcom.github.io/salesforcedx-vscode/articles/doc-link-here)'
+    '  4) Add issue links: [Issue #2490](https://github.com/forcedotcom/salesforcedx-vscode/issues/2490))'
```

Comment:
This is just supposed to be an example right?

Reply:
Yes, let me change that number so that's a bit more clear